API.md

```diff
@@ -1,17 +1,353 @@
 # Party Service API
 
-### The implemented API for this serivce can be found at /swagger/party/party-api.yaml
+### *Acceptance tests can be run to populate a local version with test data.*
 
-### CAVEAT: This page is subject to change while the Party service is being developed.
+---
 
-### Please contact the RAS development team before committing to these endpoints.
-  
-This page documents the Party service API endpoints. All endpoints return an `HTTP 200 OK` status code except where noted otherwise.
+## Business Endpoints:
 
-* `GET /parties/type/B/ref/499000011335`
-* `GET /parties/type/B/id/d826818e-179e-467b-9936-6a8603dc8b46`
+* `POST /party-api/v1/businesses`
```

Comment:
We usually have a human-readable title for each RESTful endpoint, so for example this one should be something like:
`### Create or Update a Business`

Reply:
Added